webapp/src/components/MeetingAccordion.tsx

```diff
@@ -21,26 +22,42 @@ interface MeetingAccordionProps {
 const MeetingAccordion = ({ meeting }: MeetingAccordionProps) => {
   return (
     <Accordion>
-      <AccordionSummary expandIcon={<ExpandMoreIcon />}>
-        <ListItem>
-          <MuiLink
-            component={ReactRouterLink}
-            to={`/meetings/${meeting.id}`}
-            underline="none"
-            sx={{ color: 'text.primary' }}
-            onClick={e => {
-              e.stopPropagation();
+      <AccordionSummary
+        expandIcon={
+          <IconButton>
+            <ExpandMoreIcon />
+          </IconButton>
+        }
+        sx={{
+          '&:hover': {
+            backgroundColor: 'grey.200',
+          },
+        }}
+      >
+        <MuiLink
+          component={ReactRouterLink}
+          to={`/meetings/${meeting.id}`}
+          underline="none"
+          sx={{ color: 'text.primary' }}
+          onClick={e => {
+            e.stopPropagation();
+          }}
+        >
+          <Typography variant="body1">
```

Comment:
Here's another use of the `body1` variant. I just checked the api, and it is indeed the default: https://mui.com/api/typography/

Reply:
Thanks.  I'll remove the `body1` variants where needed.